jaxrs-tck/src/main/java/ee/jakarta/tck/ws/rs/api/rs/ext/interceptor/reader/interceptorcontext/JAXRSClientIT.java

```diff
@@ -63,6 +59,15 @@ void logStartTest(TestInfo testInfo) {
   void logFinishTest(TestInfo testInfo) {
     TestUtil.logMsg("FINISHED TEST : "+testInfo.getDisplayName());
   }
+
+  @BeforeEach
```

Comment:
There is another @BeforeEach in line 53. Are they both going to be invoked by the test runner?.

Reply:
They should definitely both run, but there is no reason not to combine them IMO.

Reply:
I am more confused about removing the call of a logic for getting hostname and port and hard-coding the hostname and port here.